.github/workflows/test_thorough.bat

```diff
@@ -1,5 +1,5 @@
 set grass=%1
-set python=C:\OSGeo4W64\bin\python3
+set python=C:\OSGeo4W\bin\python3
```

Comment:
I see this is `.bat` so that's perhaps the difference, but you don't seem to need full path to Python in test_simple.sh. Maybe you can parameterize it like grass.

Reply:
Applied in 827cfac and 684c46c